model/pytorch/dffml_model_pytorch/pytorch_base.py

```diff
@@ -126,10 +126,11 @@ async def dataset_generator(self, sources: Sources):
             )
         if not y_cols:
             raise ValueError("No records to train on")
-
+        np.warnings.filterwarnings('ignore', category=np.VisibleDeprecationWarning)
```

Comment:
We can't just turn off warnings, we have to fix what it is warning us about

```suggestion
        np.warnings.filterwarnings('ignore', category=np.VisibleDeprecationWarning)
```